tables/TaQL/TableParse.cc

```diff
@@ -79,19 +81,26 @@ TableParse::TableParse()
 {}
 
 //# Constructor with given table name and possible shorthand.
-TableParse::TableParse (const Table& table, const String& shorthand)
-  : shorthand_p (shorthand),
+TableParse::TableParse (const Table& table, Int tabnr, const String& name,
+                          const String& shorthand)
+  : tabnr_p     (tabnr),
+    name_p      (name),
+    shorthand_p (shorthand),
     table_p     (table)
 {}
 
 TableParse::TableParse (const TableParse& that)
```

Comment:
Why not use the default generated compiler function?

Reply:
Removed